src/constants/lists.ts

```diff
@@ -1,3 +1,4 @@
+const UNI_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.org'
```

Comment:
we should use the ENS name `tokens.uniswap.eth`, which points to `ipns://tokens.uniswap.org`, which points at the latest version of the Uniswap token list.

Reply:
Fixed